Dockerfiles/protogen.Dockerfile

```
@@ -5,7 +5,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y
 WORKDIR /builder/
 
 RUN apt-get install --no-install-recommends -y build-essential git autoconf libtool wget unzip zlib1g-dev pkg-config cmake
-RUN wget https://github.com/protocolbuffers/protobuf/releases/download/v3.19.1/protoc-3.19.1-linux-x86_64.zip  -O protobuf.zip \
+RUN wget https://github.com/protocolbuffers/protobuf/releases/download/v3.20.1/protoc-3.20.1-linux-x86_64.zip  -O protobuf.zip \
```

```suggestion
RUN wget https://github.com/protocolbuffers/protobuf/releases/download/v21.0/protoc-21.0-linux-x86_64.zip  -O protobuf.zip \
```

maybe directly to 21 as i see 21 introduces a lot of improvements

---

Hacked